exe/bolt

```diff
@@ -10,6 +10,6 @@ begin
 rescue Bolt::CLIExit
   exit
 rescue Bolt::CLIError => e
-  warn e.message
+  warn($stderr.isatty ? "\033[1;31m#{e.message}\033[0m" : e.message)
```

Comment:
This choice of coloring may be controversial.

Reply:
I think we should not color at this level since there will be no way to disable it.

But I also think its a mistake to let so much bubble up to this level instead of handling it with the outputter and the logger. Errors should be handled through them if bolt is running normally and this should be left for extreme cases.